README.md

```diff
@@ -105,3 +103,15 @@ enough tests pass to qualify the new operating system, then either merge the pul
 the pull request with bug fixes. If they just want to retrigger the tests due to potential test flakiness, they should
 push a new commit to do so, which would include a change to publish_and_test_config.yaml with the field
 'publish_dcos_images_after: never' to avoid rebuilding identical new images.
+
+## Running Locally
```

Comment:
Run what locally?

Reply:
You can run the entire dcos-images automation locally as well - creating a dcos ami through packer, spinning up a cluster through terraform, running integration tests, accessing the cluster to test functionality, etc.

Reply:
Usually the prs are executed within ci to verify the os, docker and dc/os version is working as expected. 
When developing and fixing the scripts / tests, this explains what needs to be done to run it on the developer machine.
Side note: this can be used e.g. for provisioning a cluster / test it that is based on the different OS variants like centOS, rhel or flatcar Linux.

Reply:
Ah got it. Sorry, I thought this was a PR for the DC/OS repo and was confused.